modules/education/src/androidTest/java/edu/stanford/spezi/modules/education/video/VideoScreenTest.kt

```diff
@@ -0,0 +1,82 @@
+package edu.stanford.spezi.modules.education.video
+
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import edu.stanford.spezi.core.design.component.ComposeContentActivity
+import edu.stanford.spezi.modules.education.video.VideoScreenTest.UiStateFactory.createVideoScreenState
+import edu.stanford.spezi.modules.education.videos.Video
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@HiltAndroidTest
+class VideoScreenTest {
+
+    @get:Rule
+    val hiltRule = HiltAndroidRule(this)
+
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<ComposeContentActivity>()
+
+    private val videoScreenState = createVideoScreenState()
```

Comment:
Can't we just create a `Video` here which we pass in the screen below and avoid the factory and the custom state data class?

Reply:
Yes this would also be possible. I noted it down and will exchange it later on